ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using Task_1_7.Common;
+using System.IO;
+
+namespace ConsoleApp7
+{
+    public static class Program
+    {
+        public static FileReader fileReader = new FileReader();
+        public static void Main()
+        {
+            Console.WriteLine("Start...");
+            string filePath = Path.Combine(PathHelper.SolutionFolderPath, "Files", "Countries.txt");
+            Console.WriteLine("File Path is: " + filePath);
+            Dictionary<int, Country> dictionary = CreateDictionary(filePath);
+            if (dictionary != null)
+            {
+                dictionary = AddCountryToDictionary(dictionary);
+                dictionary = EditIsTelenorSupportedInDictionary(dictionary);
+                PrintDictionary(dictionary);
+                fileReader.PrintFile(ConvertDictionaryToList(dictionary), filePath);
+            }
+            Console.WriteLine("End...");
+            Console.ReadLine();
+        }
+
+        public static Dictionary<int, Country> CreateDictionary(string filePath)
+        {
+           if (fileReader.GetCountryListFromFile(filePath) != null)
+            {
+                int i = 0;
+                List<string> countryList = fileReader.GetCountryListFromFile(filePath);
```

Comment:
fileReader.GetCountryListFromFile(filePath) action is done twice.
Please, refactor this and put it into the try-catch clause